Sources/InternetAddress.swift

```diff
@@ -34,8 +34,8 @@ enum InternetAddress: Hashable {
         }
     }
 
-    var hashValue: Int {
-        return self.host?.hashValue ?? 0
+    func hash(into hasher: inout Hasher) {
```

Comment:
Might be nice to move this into an `extension InternetAddress: Hashable`, but not really related to this PR